### PR TITLE
Adding ir-helper.js and refactor worker.js

### PR DESCRIPTION
**Pull request by hoch**
- Added `ir-helper.js` for a simple IR buffer (Float32Array) creation. 
- Refactor `worker.js` for better readability and modularity.

### CODE REVIEW

**Comment by Dp-Goog** [comment]
Looks good so far!

**Comment by hoch** [comment]
I'll do below with a follow-up PR:

- [ ] Add a function that accepts AudioBuffer and extract an 1D Float32Array for the IR usage.

File: src/experiments/webgpuaudio/worker.js [hunk 1]
```diff
@@ -1,44 +1,71 @@
...
-    }
+// Harmful globals
+let inputQueue = null;
+let outputQueue = null;
+let atomicState = null;
+let gpuProcessor = null;
+let inputBuffer = null;
+
+// Performance metrics
+let lastCallback = 0;
```
Comment (Dp-Goog) [inline src/experiments/webgpuaudio/worker.js:14]: Should this be initialized to performance.now() instead of 0?

Comment (Dp-Goog) [reply src/experiments/webgpuaudio/worker.js:14]: Sounds good, thanks for explaining!

**Review by Dp-Goog — APPROVED** [review]

**Pull request merged**